ci/rpm/scan_daos_maldet.sh

```diff
@@ -2,32 +2,25 @@
 
 set -uex
 
-sudo yum -y install \
-  daos{,-{client,server,tests,debuginfo,devel}}-"${DAOS_PKG_VERSION}"
-
-lmd_src="maldet-current"
-lmd_tarball="maldetect-current.tar.gz"
-rm -rf "/var/tmp/${lmd_src}"
-mkdir -p "/var/tmp/${lmd_src}"
-tar -C "/var/tmp/${lmd_src}" --strip-components=1 -xf "/var/tmp/${lmd_tarball}"
-pushd "/var/tmp/${lmd_src}"
-  sudo ./install.sh
-popd
-sudo /usr/local/sbin/maldet --update-sigs
-
-fc_conf="/etc/freshclam.conf"
-if ! sudo grep -q 'ScriptedUpdates no' "$fc_conf"; then
-  sudo -E bash -c "echo \"ScriptedUpdates no\" >> \"$fc_conf\""
+source /etc/os-release
+: "${ID_LIKE:=unknown}"
+: "${ID:=unknown}"
+version="${VERSION_ID%%.*}"
+if [[ $ID_LIKE == *rhel* ]]; then
+  distro=el
 fi
-: "${JOB_URL:=${JENKINS_URL}job/clamav_daily_update/}"
-if ! sudo grep -q "$JENKINS_URL" "$fc_conf"; then
-  clam_url="${JOB_URL}lastSuccessfulBuild/artifact/download/clam"
-  sudo -E bash -c "echo \"PrivateMirror ${clam_url}\" >> \"$fc_conf\""
+if [[ $ID == *leap* ]]; then
+  distro=leap
 fi
-sudo freshclam
-rm -f /var/tmp/clamscan.out
-rm "/var/tmp/${lmd_tarball}"
-rm -rf "/var/tmp/${lmd_src}"
+
```

Comment:
Is this all something that needs to be in a common functions file and sourced.  We seem to be repeating it bit.

Reply:
Looks like this can possibly be part of ci/parse_ci_env.sh in the future.
Looks like ci/parse_ci_env.sh needs to be updated to specify a target of "el8" instead of "centos8" though, but not planning on changing and testing that as part of this PR.